python/sglang/srt/model_executor/forward_batch_info.py

```diff
@@ -340,7 +340,7 @@ def init_new(
             ret.extend_prefix_lens = torch.tensor(
                 batch.extend_prefix_lens, dtype=torch.int32
             ).to(device, non_blocking=True)
-            if model_runner.server_args.attention_backend != "torch_native":
+            if support_triton(model_runner.server_args.attention_backend):
```

Comment:
@yanbing-j we are very likely to provide an optimized implementation e.g.  `compute_position_cpu` when the cpu kernels are compiled.